tests/test_handlers.py

```diff
@@ -25,7 +25,8 @@
 LOG_DIR: Path = Path("unittest-logs")
 
 
-class DtStreamHandler(logging.StreamHandler):
+# TODO: revisit type ignore if minimum python version increased
```

Comment:
Is there a version where this is no longer necessary or do we just need to revisit all type ignores every time we increase the minimum Python version?